source/configure/calls-deployment.md

```diff
@@ -219,15 +231,16 @@ Mattermost Calls can function in air-gapped environments. Exposing Calls to the
 Calls performance primarily depends on:
 
 - **CPU resources**: More participants require more processing power
-- **Network bandwidth**: Both incoming and outgoing traffic increases with participant count
+- **Network bandwidth**: Both incoming and outgoing traffic increases with participant count. Due to the nature of the service, the bottleneck is always going to be the outgoing/egress path
 - **Active speakers**: Unmuted participants require significantly more resources 
+- **Presenters**: Screen sharing participants require even more resources than active speakers 
```

Comment:
Continuing from [this comment](https://github.com/mattermost/docs/pull/7929/commits/0f32682c02c97e1cff4279a78df23dc1f7be20b0#r2195453292)

How's this extra bullet @streamer45